src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsIssuesListener.php

```diff
@@ -38,109 +39,35 @@ public function onIssueAfterCreate(Event $event)
 		if ($arguments['action'] === 'opened')
 		{
 			// Add a "no code" label
-			$this->addCodelabel($arguments['hookData'], $arguments['github'], $arguments['logger'], $arguments['project']);
+			$this->checkNoCodelabel($arguments['hookData'], $arguments['github'], $arguments['logger'], $arguments['project'], $arguments['table']);
 		}
 	}
 
 	/**
 	 * Adds a "No Code Attached Yet" label
 	 *
-	 * @param   object  $hookData  Hook data payload
-	 * @param   Github  $github    Github object
-	 * @param   Logger  $logger    Logger object
-	 * @param   object  $project   Object containing project data
+	 * @param   object       $hookData  Hook data payload
+	 * @param   Github       $github    Github object
+	 * @param   Logger       $logger    Logger object
+	 * @param   object       $project   Object containing project data
+	 * @param   IssuesTable  $table     Table object
 	 *
 	 * @return  void
 	 *
 	 * @since   1.0
 	 */
-	protected function addCodelabel($hookData, Github $github, Logger $logger, $project)
+	protected function checkNoCodelabel($hookData, Github $github, Logger $logger, $project, IssuesTable $table)
 	{
 		// Set some data
-		$codeLabel    = 'No Code Attached Yet';
-		$addLabels    = array();
-		$codeLabelSet = false;
+		$label      = 'No Code Attached Yet';
+		$labels     = array();
+		$labelIsSet = $this->checkLabel($hookData, Github $github, Logger $logger, $project, IssuesTable $table, $label);
```

Comment:
You should remove type hinting here. And also do not forget to remove `$table` as it is not used.